kyuubi-server/src/main/scala/org/apache/kyuubi/operation/KyuubiOperation.scala

```diff
@@ -158,4 +160,10 @@ abstract class KyuubiOperation(opType: OperationType, session: Session)
   }
 
   override def shouldRunAsync: Boolean = false
+
+  override def setState(newState: OperationState): Unit = {
+    super.setState(newState)
+    MetricsSystem.tracing(
+      _.markMeter(MetricRegistry.name(OPERATION_STATE_RATE, newState.toString.toLowerCase)))
```

Comment:
do we need toLowerCase? seems we are not doing this for OperationTypes

Reply:
Yes, we need. `OpertationType` is also use `toLowerCase`. Use low case to make them consistent.